### PR TITLE
fix: wrong options name for exclude_tags_regex in warn function

### PR DESCRIPTION
**Pull request by ldez**

### CODE REVIEW

**Comment by ldez** [comment]
any remarks ?

**Comment by Lucashuang0802** [comment]
@ldez if you search `warn_if_nonmatching_regex`, it is being used in `filter_exact_tags` with `:exclude_tags`. Therefore, I don't think it is an error. 😃

File: lib/github_changelog_generator/generator/generator_tags.rb [hunk 1]
```diff
@@ -183,9 +183,9 @@ def filter_exact_tags(all_tags)
     end
 
     def warn_if_nonmatching_regex(all_tags)
-      unless all_tags.map { |t| t["name"] }.any? { |t| options[:exclude_tags] =~ t }
+      unless all_tags.map { |t| t["name"] }.any? { |t| /#{options[:exclude_tags_regex]}/ =~ t }
```
Comment (Lucashuang0802) [inline lib/github_changelog_generator/generator/generator_tags.rb]: @ldez I recently started building up my Ruby skill. I am wondering if double quote makes it clear?

**Comment by ldez** [comment]
@Lucashuang0802 it's also used in:
```ruby
     def filter_tags_with_regex(all_tags, regex)
       warn_if_nonmatching_regex(all_tags)
       all_tags.reject { |tag| regex =~ tag["name"] }
     end
```

**Comment by olleolleolle** [comment]
I believe this has been fixed in `master`. See this diff in the conflict above:

```diff
<<<<<<< patch-1
    def warn_if_nonmatching_regex(all_tags)
      unless all_tags.map { |t| t["name"] }.any? { |t| /#{options[:exclude_tags_regex]}/ =~ t }
        Helper.log.warn "Warning: unable to reject any tag, using regex "\
                        "#{options[:exclude_tags_regex].inspect} in --exclude-tags-regex "\
                        "option."
=======
    def warn_if_nonmatching_regex(all_tags, regex, regex_option_name)
      unless all_tags.map { |t| t["name"] }.any? { |t| regex =~ t }
        Helper.log.warn "Warning: unable to reject any tag, using regex "\
                        "#{regex.inspect} in #{regex_option_name} option."
>>>>>>> master
```

**Comment by olleolleolle** [comment]
Thank you for your contribution! I'll close this issue as fixed.

**Pull request closed without merging**